docs/releasenotes/10.4.0.rst

```diff
@@ -42,6 +42,14 @@ The ``hints`` parameter in :py:meth:`~PIL.ImageDraw.getdraw()` has been deprecat
 API Changes
 ===========
 
+ImageFont.truetype
+^^^^^^^^^^^^^^^^^^
+
+:py:function:`~PIL.ImageFont.truetype` now searches for fonts in the directory
```

Comment:
```suggestion
:py:func:`~PIL.ImageFont.truetype` now searches for fonts in the directory
```
You can see at https://pillow--8135.org.readthedocs.build/en/8135/releasenotes/10.4.0.html#imagefont-truetype that this isn't rendering correctly as is.

Reply:
Oh thanks, didn't notice that. Let me commit that change.